kubernetes/cray-kafka-operator/values.yaml

```diff
@@ -7,78 +7,92 @@ strimzi-kafka-operator:
     - sma
 
   image:
-    repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+    registry: artifactory.algol60.net
+    repository: csm-docker/stable/quay.io/strimzi
```

Comment:
Not sure if it will break the image reference by changing from a single "repository" to two separated "registry" and "repository". If it works, then great. Similar comments for a few other references below.

Reply:
Since we're using an upstream module we're at the mercy of how it works. If you leave out the registry bit then it'll prepend quay.io to whatever's in repository.

Reply:
The reference to the image is now:

`image: {{ default .Values.defaultImageRegistry .Values.image.registry }}/{{ default .Values.defaultImageRepository .Values.image.repository}}/{{ .Values.image.name }}:{{ default .Values.defaultImageTag .Values.image.tag }}`

Reply:
Great. Thanks for clarifying!